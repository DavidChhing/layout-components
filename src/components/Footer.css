footer {
  font-size: 55px;
  text-align: center;
  padding: 1rem;
  text-align: center;
}
