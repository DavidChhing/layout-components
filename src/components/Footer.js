import Component from './Component';
import styles from './Footer.css';

export default class Footer extends Component {
  render() {
    const footerContent = `${this.props.footerContent}`;

    return `<footer class="${styles.footer}">
    <h3>Stuff About Me</h3>
    ${footerContent}
    </footer>`;
  }
}
