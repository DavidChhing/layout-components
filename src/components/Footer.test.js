import Footer from './Footer';

describe('Footer component', () => {
  it('should be a footer', () => {
    const footer = new Footer({
      footerContent: 'I like pizza and sushi; however, I like coding more.',
    });

    expect(footer.render()).toMatchSnapshot();
  });
});
