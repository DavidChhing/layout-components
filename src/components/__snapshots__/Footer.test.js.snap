// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component should be a footer 1`] = `
"<footer class=\\"footer\\">
    <h3>Stuff About Me</h3>
    I like pizza and sushi; however, I like coding more.
    </footer>"
`;
